frontend/src/app/AboutDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  AboutModal,
+  Bullseye,
+  Spinner,
+  TextContent,
+  TextList,
+  TextListItem,
+} from '@patternfly/react-core';
+import { ODH_LOGO, ODH_PRODUCT_NAME } from '~/utilities/const';
+import { useUser, useClusterInfo } from '~/redux/selectors';
+import { useAppContext } from '~/app/AppContext';
+import useFetchDsciStatus from '~/concepts/areas/useFetchDsciStatus';
+
+import './AboutDialog.scss';
+
+const RhoaiAboutText = `Red Hat® OpenShift® AI (formerly Red Hat OpenShift Data Science) is a flexible, scalable MLOps platform for data scientists and developers of artificial intelligence and machine learning (AI/ML) applications. Built using open source technologies, OpenShift AI supports the full lifecycle of AI/ML experiments and models, on premise and in the public cloud.`;
+const RhoaiDefaultReleaseName = `OpenShift AI`;
+
+const OdhAboutText = `Open Data Hub is an open source AI platform designed for the hybrid cloud. The community seeks to bridge the gap between application developers, data stewards, and data scientists by blending the leading open source AI tools with a unifying and intuitive user experience. Open Data Hub supports the full lifecycle of AI/ML experiments and models.`;
+const OdhDefaultReleaseName = `Open Data Hub operator`;
+
+interface AboutDialogProps {
+  onClose: () => void;
+  productName?: string; // Used only for testing
+}
+
+const AboutDialog: React.FC<AboutDialogProps> = ({ onClose, productName = ODH_PRODUCT_NAME }) => {
+  const { isAdmin } = useUser();
+  const { operatorSubscriptionStatus, isRHOAI } = useAppContext();
+  const { serverURL } = useClusterInfo();
+  const [dsciStatus, loadedDsci, errorDsci] = useFetchDsciStatus();
+  const loading = !errorDsci && !loadedDsci;
+
+  return (
+    <AboutModal
+      className="odh-about-dialog"
+      isOpen
+      onClose={onClose}
+      brandImageSrc={`${window.location.origin}/images/${ODH_LOGO}`}
+      brandImageAlt={`${productName} logo`}
+      productName={productName}
+      aria-label={productName}
+      data-testid="odh-about-dialog"
+    >
+      <TextContent style={{ paddingBottom: 48 }}>
+        <h4>About</h4>
+        <p data-testid="about-text">{isRHOAI ? RhoaiAboutText : OdhAboutText}</p>
+      </TextContent>
+      <TextContent>
+        <div style={{ position: 'relative' }}>
+          {loading ? (
+            <Bullseye style={{ position: 'absolute', width: '100%' }}>
+              <Spinner size="xl" />
+            </Bullseye>
+          ) : null}
+          <TextList component="dl" style={{ visibility: loading ? 'hidden' : 'visible' }}>
```

Comment:
Why a visibility style instead of part of the loading else clause for conditional rendering this section?
Is this done just to establish some white space for positioning the spinner mid dialog?

Reply:
Yes, it is to keep the dialog from changing size once the data is loaded.